.github/workflows/package.yml

```diff
@@ -0,0 +1,36 @@
+name: 📦 Package
+
+on:
+  workflow_call:
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: sentry-godot-gdextension
+          path: artifact/
+
+      - name: Prepare artifact
+        shell: bash
+        run: |
+          # * Fix crashpad_handler permissions, workaround for https://github.com/actions/upload-artifact/issues/38
```

Comment:
Ideally this goes into a script so it's easier to test locally and debug stuff

Reply:
It relies on actually downloading an artifact - something that is a merged product of a number of builds. Testing it locally would be cumbersome, since it's not enough to produce a single build for this.